src/components/TileGrid/SeniorTile.tsx

```diff
@@ -35,19 +37,25 @@ export function SeniorTile({
         setSeniorPatch(senior.id);
         setShowAddSeniorPopUp(true);
       },
+      color: "#22555A",
+      icon: <FontAwesomeIcon icon={faPencil} />,
     },
     {
       name: "Delete",
       onClick: (e) => {
         e.stopPropagation();
         e.preventDefault();
-        deleteSenior({ seniorId: senior.id });
+        fetch(`/api/senior/${senior.id}`, {
```

Comment:
I think we inadvertently re-introduced the old code here during rebase. This should be `deleteSenior({ seniorId: senior.id })`